Assets/Mapzen/VectorData/Formats/GeoJsonTile.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using SimpleJSON;
+
+namespace Mapzen.VectorData.Formats
+{
+    public class GeoJsonTile : Tile
+    {
+        public delegate Point LocalCoordinateProjection(double longitude, double latitude);
+
+        public static LocalCoordinateProjection LocalCoordinateProjectionForTile(TileAddress address)
+        {
+            return delegate(double longitude, double latitude)
+            {
+                var lngLat = new LngLat(longitude, latitude);
+                var meters = lngLat.ToMercatorMeters();
+                var origin = address.GetOriginMercatorMeters();
+                var scale = address.GetSizeMercatorMeters();
+                var x = (meters.x - origin.x) / scale;
+                var y = (origin.y - meters.y) / scale;
+                return new Point((float)x, (float)y);
```

Comment:
Good call, we definitely want to make the coordinates consistent between formats. However after thinking about this a while I think the GeoJSON coordinates are actually more correct here. The current implementation of `MercatorMeters` and `TileAddress` dictates that `address.GetOriginMercatorMeters()` is the _Northwest_ corner of the tile area, so if +Y is North in local coordinates, then geometry in a tile _should_ be < 0 in Y.

We can resolve this either by defining the interior coordinates for a tile as { 0...1, -1...0 } and adjusting the MVT coordinates to match, or changing the semantics of `MercatorMeters` and the tile address origin.